### DISCUSSION TITLE
GSoC collection of tasks

### DISCUSSION THREAD

**Discussion by niccokunzmann** [collective/icalendar#663] [General]
I could be possible that we allow students to work on `icalendar` for a Google Summer of Code or other funded activities.
Therefore, I would like to create a list of issues that are useful to make the case that there is enough valuable work that can be done on this project.

## Proposals

The following standards should be tested and included, see #589:

- #652
- #653
- #654
- #655
- #656
- #657
- #658
- vcard format? https://www.rfc-editor.org/rfc/rfc6350.html
- jcal: https://github.com/collective/icalendar/discussions/702

I would like to have some usability improvements included:

- #662
- #610
- #496
- #395
- #315
- #224
- #158
- #124
- https://github.com/collective/icalendar/issues/722

Bugs:

- generally all up for grabs, especially to get to know the project
- #349
- #336

Code quality:

- #604 #444
- #582

Documentation:

- #626 
- #443

External:

- https://github.com/niccokunzmann/python-recurring-ical-events/issues/193

This is an extensible list and I believe that we can keep someone busy not only fixing bugs but also improving usability and documentation.

**Comment by stevepiercy** [comment]
The Plone Foundation and the GSoC categories would be a good place to start to get an organization to lean on, if you need one.

- https://community.plone.org/tag/gsoc
- https://community.plone.org/c/plone-foundation/47
